doc/TUTORIALS_CN.md

```diff
@@ -893,34 +1169,39 @@ dsets/
 │   ├── reg_test_x.csv
 │   ├── reg_test_y.csv
 │   ├── reg_train_x.csv
-│   └── reg_train_y.csv
+│   ├── reg_train_y.csv
+│   ├── mnist_test_x.csv
+│   └── mnist_test_y.csv
 ├── P0
 │   ├── cls_test_x.csv
 │   ├── cls_test_y.csv
 │   ├── cls_train_x.csv
 │   ├── cls_train_y.csv
 │   ├── reg_test_x.csv
-│   └── reg_train_x.csv
+│   ├── reg_train_x.csv
+│   └──	mnist_train_x.csv
 ├── P1
 │   ├── cls_test_x.csv
 │   ├── cls_train_x.csv
 │   ├── reg_test_x.csv
 │   ├── reg_test_y.csv
 │   ├── reg_train_x.csv
-│   └── reg_train_y.csv
+│   ├── reg_train_y.csv
+│   └──	mnist_train_x.csv
```

Comment:
ditto

Reply:
![image](https://user-images.githubusercontent.com/24633447/124948909-b3dd7d80-e043-11eb-9835-124918803b36.png)
In the commits, there is the extra space, but, not in the document.